server.js

```diff
@@ -66,6 +67,20 @@ app.put('/api/foods/:id', function(request, response) {
   })
 })
 
+app.get('/api/diary', function(request, response) {
```

Comment:
@case-eee - an alternative if we can't get this working is to update this API endpoint to just take the meal id, and then when we load the food log, we make an API call for each meal to get their foods. That way when we delete or update a food, instead of getting all the information for all foods and meals, just just get the foods that need to be re-populated for the the meal that was updated.